### ISSUE TITLE
Should TickThreading perhaps have a better name?

### ISSUE THREAD

**Issue by Puremin0rez** [MinimallyCorrect/TickThreading#107]
It seems like this is becoming the ultimate Forge improvement suite and slightly varying away from just threading things, including many performance improvements and great features like the new anti cheat stuff.

I myself can't think of a name that would fit, but it seems like TT is basically the Spigot of Bukkit at this point :)

**Comment by LunNova** [comment]
I'm not very good at names. If anyone has any sensible suggestions, please reply here.

**Comment by LunNova** [comment]
`<BitVector> BTF - BetterThanForge`

**Comment by Puremin0rez** [comment]
That's actually quite and interesting name...

That looks like an IRC message, does TT have an IRC?

**Comment by LunNova** [comment]
No, but it is semi-regularly discussed in the #ftb-craft channel on espernet, and now it seems sometimes also #bukkitforge for questions related to using TT in conjunction with it.

edit: And also #n-builds for build announcements.

**Comment by Puremin0rez** [comment]
Here are some suggestions I've gotten from others and myself:
OptiForge
Forge++
ForgePerf
Forge Enhancement Suite

**Comment by ghost** [comment]
Forge Server Enhancement Suite gets my vote.

**Comment by LunNova** [comment]
FES > FSES, because FES is closer to ![fez](http://4.bp.blogspot.com/_SxHRFN5zD8c/R9PYPMiGOOI/AAAAAAAAAFI/ZJF4LmBHW24/s400/thumbsup.php.jpg).

**Comment by Puremin0rez** [comment]
I also vote for Forge Enhancement Suite! FES sounds neat! :+1:

**Comment by GUIpsp** [comment]
FES is too much like RES

**Comment by LunNova** [comment]
Closing this, mainly because the name's kinda' known now, and I'm too lazy to change it. The name's "good enough".  Maybe.

**Comment by LunNova** [comment]
Looking for input on this one again. Fezzes are cool, but it probably isn't a great name for googling as it's a common word.

**Comment by Unnoen** [comment]
TickEnhancement
ChunkEnhancement
NullTicks
ChunkFaster
TooManyTicks
NotEnoughTicks
MyTPSisBiggerThanYours

**Comment by tenten8401** [comment]
MultiplexedMC: Fixing what mojang didn't.

**Comment by cpypcy** [comment]
Tick Threading - The Way It's Meant To Be Hosted!
Tick Threading - Server performance over 9000!
Tick Threading - My mob spawner brings 10 000 cows to the yard and i'm like WOW DUDE THERE'S NO LAG!
Tick Threading - Putting your multicore CPU through bootcamp
Tick Threading - Just one more thread...
Tick Threading - I bet i can make this MMO!
Tick Threading - Your 1000 pistons have no power here!
Tick Threading - Valhalla of Viking servers!
Tick Threading - Make servers great again!
Tick Threading - #212 quote #211 funny #214 found #213 not

**Comment by cosmicdan** [comment]
Why not "MulticoreTweaks"? No fluff, specific, clear objective.

**Comment by Bucky420** [comment]
Threads = String = Harp
or MCHarp
